config/config.go

```diff
@@ -572,6 +572,30 @@ func (c *Config) NeedToGenerateERImages() bool {
 	return true
 }
 
+func (c *Config) detectShowColumnsForER(s *schema.Schema) error {
+	if c.ER.ShowColumnTypes == nil {
+		return nil
+	}
+
+	if !c.ER.ShowColumnTypes.Related {
+		return errors.New("er.showColumnTypes: must be true at least one")
+	}
+
+	for _, t := range s.Tables {
+		for _, cc := range t.Columns {
+			// related
+			if t.Name == "a2" {
+				fmt.Printf("%#v\n", cc)
+			}
```

Comment:
Did you forget to turn off the debug output?

Reply:
oh... Thank you!